src/PFXImportPowershell/README.md

```diff
@@ -69,17 +106,40 @@ Import-IntunePublicKey -ProviderName "<ProviderName>" -KeyName "<KeyName>" -File
 ```
 
 ## Authenticate to Intune
+
+### User Authentication with interactive login
+
+1. Authenticate as the account administrator (using the admin UPN) to Intune. Specify the AdminUserName on the command line, but not the AdminPassword.  An interactive login dialog will appear.
+```
+Set-IntuneAuthenticationToken -AdminUserName "<Admin-UPN>" 
+```
+2. Make sure the call Remove-IntuneAuthenticationToken to clear the token cache when all interation with Intune is complete.  Close the PowerShell session to remove credentials cached by the interactive browser.
+
+
+### User authentication with non-interactive login
+
+Prerequisite: to use this option, enable “Allow public client flows” setting on the app registration "Authentication" page.
+
 1. Optionally, create a secure string representing the account administrator password.
 ```
 $secureAdminPassword = ConvertTo-SecureString -String "<admin password>" -AsPlainText -Force
 ```
-2. Authenticate as the account administrator (using the admin UPN) to Intune. If the password is not provided a login dialog will appear.
+2. Authenticate as the account administrator (using the admin UPN) to Intune.  Provide both the user name and the password on the command line.
 ```
 Set-IntuneAuthenticationToken -AdminUserName "<Admin-UPN>" [-AdminPassword $secureAdminPassword]
 ```
 3. Make sure the call Remove-IntuneAuthenticationToken to clear the token cache when all interation with Intune is complete.
 
-## Set up userPFXCertifcate object (scenario: encrypting password from a location that has acccess to the private key in the key store) 
+### Application client secret authentication
+
+Prerequisite: create the client secret in the app registration and configure the IntunePfxImport.psd1 file.
```

Comment:
Is there azure docs we can link to here?

Reply:
will add a link in the "Configure a Microsoft Azure App Registration" to the azure app registration quick start guide.